textbook/13/chapter12-glossary.md

```diff
@@ -0,0 +1,53 @@
+## Model
+A set of rules that describe how data are generated.
+
+## Consistency
+Evaluation of the plausibility that data was generated from a particular model. In other words, when evaluating a hypothetical scenario, we may ask "Is this outcome consistent with the proposed model?"
+
+## Hypothesis
+In statistics, a hypothesis is defined as a null or alternative view of how data were generated from a model.
+
+## Null Hypothesis
+The default view (or hypothesis) that is generally believed to be true.
+
+## Alternative Hypothesis
+A view (or hypothesis) that is opposite of the null hypothesis.
+
+## Test statistic
+A summary measure of the data that we use to investigate the consistency of a model. The choice of a test statistic depends on the specific hypotheses we are investigating. For example, the difference in means, medians, or standard deviations could all be used as test statistics.
+
+## P-value
+The chance, under the null hypothesis, that the test statistic is equal to the observed value or is further in the direction of the alernative hypothesis.
+
+## Two Sample Test
+A hypothesis test performed to investigate whether there is a difference between two random samples or groups. This is also called A/B testing as we can refer to our two groups as Group A and Group B.
+
+## A/B Test
+A hypothesis test performed to investigate whether there is a difference between two random samples or groups (e.g., Group A and Group B). This is also called a Two Sample Test.
+
+## Boxplot
```

Comment:
You should be able to take out boxplot and IQR because we cover those in data viz